### ISSUE TITLE
How to check for "recordness"

### ISSUE THREAD

**Issue by nyaray** [extend/sheriff#18]
How do I check if a value is an instance of a record? A function is clearly generated since I get a warning about not using it, but I simply can't figure out how to call it via sheriff:check/2 calls.

I didn't really know who to ask, so I figured that an issue would be suitable (since it really isn't documented from what I've read). It would be nice if the answer to this question was included in the documentation for future users wanting to validate values as records and not just types.

**Comment by essen** [comment]
Hm that's a good point. There's a workaround, but I should check if we can do anything better:

``` erlang
-type myrec() :: #myrec{}.

doit() ->
    sheriff:check(42, myrec).
```

I'll get back to you on proper explanations and solutions.

**Comment by essen** [comment]
Records have been fixed, there was many issues: 166f28a1eac2bdf557c4d073f39038c7cb587477.

However I don't really feel like more is needed, though it's definitely worth adding to the README, done here: e10f8ac999f3755525f41eb42f7b578f83844351.

Thanks for the report!

**Comment by nyaray** [comment]
Awesome, thanks for picking it up again :). I agree, you probably don't need more functionality, that would be another lib/app.
